backend/controllers/authController.js

```diff
@@ -1,10 +1,19 @@
+const jwt = require('jsonwebtoken');
 const User = require("../models/User");
 
 const handleErrors = (err) => {
   if (err.code === 11000) {
     return { email: 'Email is already registered' };
   }
-return  Object.keys(err.errors).reduce((acc, key) => {
+
+  const errorKeys = Object.keys(err.errors)
+  if (errorKeys.length){
+    return {
+      message: err
+    }
+  }
```

Comment:
added this so the database doesn't break if there are errors that don't have keys